database/mongoDb.js

```diff
@@ -39,47 +39,31 @@ let Rating = mongoose.model('Rating', ratingSchema);
 const getAllReviews = () => {
   return new Promise ((resolve, reject) => {
```

Comment:
This works, but you're essentially wrapping a promise in a promise. You can accomplish the same thing by returning Review.find().exec(). Consider replacing mongoose.Promise with bluebird or another promise library

Reply:
I needed that push. I just refactored a lot... glad it still works!

Reply:
FYI Jason made me aware that the most recent version of mongoose doesn't require you to replace the promise library